src/main/java/org/opensearch/performanceanalyzer/http_action/config/PerformanceAnalyzerClusterConfigAction.java

```diff
@@ -160,6 +160,7 @@ public List<ReplacedRoute> replacedRoutes() {
     @Override
     protected RestChannelConsumer prepareRequest(final RestRequest request, final NodeClient client)
             throws IOException {
+        request.param("verbose");
```

Comment:
Why is this needed?

Reply:
It is required by the RestRequest class to mark "verbose" as a consumed parameter. Else it rejects all unknown params.

Reply:
See comment on L234 - https://github.com/opensearch-project/performance-analyzer/blob/066cef88b29b5ddfa108ce8520f3b45436ac0866/src/main/java/org/opensearch/performanceanalyzer/http_action/config/PerformanceAnalyzerResourceProvider.java#L230-L238

Yes, this method is primarily meant to fetch the value of the `verbose` param which correspondingly marks it as "consumed." Since we do not care about the actual param value, there is no need to store the output of this call.